### PR TITLE
(docs) Comments in outgoing app

### PR DESCRIPTION
**Pull request by michalc**

### CODE REVIEW

File: core/app/app_outgoing.py [hunk 1]
```diff
@@ -81,6 +81,12 @@
 
 
 async def run_outgoing_application():
+    """Indefinitely poll paginated feeds and ingest them into Elasticsearch
```
Comment (WillTaylor22) [inline core/app/app_outgoing.py]: What specifically does this method do to help achieve this? All of the following methods have similar definitions.

File: core/app/app_outgoing.py [hunk 2]
```diff
@@ -133,6 +146,13 @@
 
 
 async def ingest_feeds(context, feeds):
+    """Create a task that polls feeds and ingests them into Elasticsearch
```
Comment (WillTaylor22) [inline core/app/app_outgoing.py]: Linked to above, what specifically does this function do, as different from run_outgoing_application or create_outgoing_application?

File: core/app/app_outgoing.py [hunk 3]
```diff
@@ -219,6 +291,10 @@ def feed_unique_ids(feeds):
 
 
 async def ingest_page(context, ingest_type, feed, activity_index_names, objects_index_names, href):
+    """Fetch a single page of data from a feed and ingest into Elasticsearch
```
Comment (WillTaylor22) [inline core/app/app_outgoing.py]: How is this different to get_feed_contents?

File: core/app/app_outgoing.py [hunk 4]
```diff
@@ -255,6 +331,16 @@ def feed_unique_ids(feeds):
 
 
 async def get_feed_contents(context, href, headers):
+    """Fetch a single page of data from a feed, returning it as bytes
```
Comment (WillTaylor22) [inline core/app/app_outgoing.py]: How is this different to ingest_page?

File: core/app/app_outgoing.py (hunk 3 above)
Comment (WillTaylor22) [reply core/app/app_outgoing.py]: To be more specific, it says it does fetching and then ingesting. While get_feed_contents does fetching only. It's not clear who's responsible for what.

File: core/app/app_outgoing.py (hunk 1 above)
Comment (michalc) [reply core/app/app_outgoing.py]: I have added some more detail

**Pull request merged**